ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml

```diff
@@ -1667,31 +1728,29 @@ objects:
 
           gcloud deployment-manager deployments create ${CLUSTER_NAME}-security --config 03_security.yaml
 
-          export MASTER_SA=${INFRA_ID}-m@${PROJECT_NAME}.iam.gserviceaccount.com
-          gcloud projects add-iam-policy-binding ${PROJECT_NAME} --member "serviceAccount:${MASTER_SA}" --role "roles/compute.instanceAdmin"
-          gcloud projects add-iam-policy-binding ${PROJECT_NAME} --member "serviceAccount:${MASTER_SA}" --role "roles/compute.networkAdmin"
-          gcloud projects add-iam-policy-binding ${PROJECT_NAME} --member "serviceAccount:${MASTER_SA}" --role "roles/compute.securityAdmin"
-          gcloud projects add-iam-policy-binding ${PROJECT_NAME} --member "serviceAccount:${MASTER_SA}" --role "roles/iam.serviceAccountUser"
-          gcloud projects add-iam-policy-binding ${PROJECT_NAME} --member "serviceAccount:${MASTER_SA}" --role "roles/storage.admin"
+          export MASTER_SERVICE_ACCOUNT=$(gcloud iam service-accounts list | grep "^${INFRA_ID}-master-node " | awk '{print $2}')
+          export WORKER_SERVICE_ACCOUNT=$(gcloud iam service-accounts list | grep "^${INFRA_ID}-worker-node " | awk '{print $2}')
```

Comment:
Same comment as https://github.com/openshift/installer/pull/3270#discussion_r391296306

Reply:
Can we fix this in a follow up PR after we resolve the discussion? I'd like to go ahead and merge this to unblock the other PRs. This method was being used before this change, I simply consolidated two different methods of calculating these values and added `$()` instead of ` `` `.

Reply:
Followed up in #7620